apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableLikeToContainInOrderOnlyCreatorSamples.kt

```diff
@@ -5,6 +5,19 @@ import ch.tutteli.atrium.api.verbs.expect
 import kotlin.test.Test
 
 class IterableLikeToContainInOrderOnlyCreatorSamples {
+    @Test
+    fun value(){
+        expect(listOf("A")).toContain.inOrder.only.value("A")
+
+        fails { // because subject list does not contain expected value
+            expect(listOf("B")).toContain.inOrder.only.value("A")
+        }
+
+        fails { // because subject list contains multiple elements
+            expect(listOf("A","A")).toContain.inOrder.only.value("A")
+        }
```

Comment:
can you please add another example with `A` and `B` where it fails because the order is wrong

Reply:
Yes, I'll do. (sorry for my delayed reaction. I did not realize this comment when i wrote last comment.)